include/control.h

```diff
@@ -70,15 +71,22 @@ class Config {
 
 	~Config();
 
-	Section_line * AddSection_line(char const * const _name,void (*_initfunction)(Section*));
-	Section_prop * AddSection_prop(char const * const _name,void (*_initfunction)(Section*),bool canchange=false);
-	
-	Section* GetSection(int index);
-	Section* GetSection(std::string const&_sectionname) const;
+	Section_prop *AddEarlySection_prop(const char *name,
+	                                   SectionFunction func,
+	                                   bool canchange = false);
+
+	Section_line *AddSection_line(char const *const _name, SectionFunction func);
+
+	Section_prop *AddSection_prop(char const *const _name,
+	                              SectionFunction func,
+	                              bool canchange = false);
```

Comment:
Yes, `canchange` is one of those subtle runtime behaviour modifiers allowing `*_Init()` and `*_Destroy()` functions to be re-called at runtime, after the initial setup phase. You can see it in the `if ... `:

``` c++
void Section::ExecuteInit(bool init_all)
{
	for (const auto &fn : early_init_functions)
		if (init_all || fn.canchange)
			fn.function(this);
}

I've renamed it to be `changeble_at_runtime`, because that's actually what it means.  
A tad verbose, but not too painful I think.